RecoEgamma/EgammaPhotonProducers/python/photons_cfi.py

```diff
@@ -150,3 +150,11 @@
     RecHitFlagToBeExcludedEE = multi5x5BasicClustersCleaned.RecHitFlagToBeExcluded,
     RecHitSeverityToBeExcludedEE = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
 )
+
+from Configuration.ProcessModifiers.egamma_lowPt_exclusive_cff import egamma_lowPt_exclusive
+egamma_lowPt_exclusive.toModify(photons,
+                           minSCEtBarrel = 1.0, #default 10
+                           minSCEtEndcap = 1.0) #default 10
+egamma_lowPt_exclusive.toModify(islandPhotons,
+                           minSCEtBarrel = 1.0, #default 5
+                           minSCEtEndcap = 1.0) #default 15
```

Comment:
this is not a part of the regular pp reco.
Why is this modified?

Reply:
ok, we will not change the island Photon thresholds (updating it in the next commit).